docker-entrypoint.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -e
 
+# If long & short hostnames are not the same, use long hostnames
+if [ $(hostname) != $(hostname -s) ]; then
+	export RABBITMQ_USE_LONGNAME=true
+fi
+
+if [ ${RABBITMQ_SSL_CERT_FILE:+x} -a ${RABBITMQ_SSL_KEY_FILE:+x} -a ${RABBITMQ_SSL_CA_FILE:+x} ]; then
+	use_ssl="yes"
```

Comment:
The rest of this file uses `camelCase` for local vars -- let's update this one to something like `useSsl` or even just `ssl`, and make sure to initialize it to the empty string before the `if`, ie:
```bash
ssl=
if [ "$RABBITMQ_SSL_CERT_FILE" -a "$RABBITMQ_SSL_KEY_FILE" -a "$RABBITMQ_SSL_CA_FILE" ]; then
	ssl=1
fi
```